internal/install/static_snapshot_yml.go

```diff
@@ -21,11 +21,13 @@ services:
     - "xpack.license.self_generated.type=trial"
     - "xpack.security.enabled=true"
     - "xpack.security.authc.api_key.enabled=true"
+    - "xpack.security.authc.realms.file.file1.order=0"
     - "ELASTIC_PASSWORD=changeme"
     - "script.context.template.max_compilations_rate=unlimited"
     - "script.context.ingest.cache_max_size=2000"
     - "script.context.processor_conditional.cache_max_size=2000"
     - "script.context.template.cache_max_size=2000"
+    command: bash -c "bin/elasticsearch-users useradd elastic -r superuser -p changeme | /usr/local/bin/docker-entrypoint.sh eswrapper"
```

Comment:
Looks like the error is complaining about trying to add the `elastic` user which is a built-in user so it already exists. What happens if you remove this line?

Reply:
I think that was a similar problem. I can give it a try.

Reply:
Unfortunately same problem: https://beats-ci.elastic.co/job/Ingest-manager/job/elastic-package/job/PR-240/4/artifact/build/elastic-stack-dump/stack/logs/elasticsearch.log